firefox-ios/Client/Frontend/Browser/TabPrintPageRenderer.swift

```diff
@@ -5,52 +5,51 @@
 import Foundation
 import Common
 
-private struct PrintedPageUX {
-    static let PageInsets = CGFloat(36.0)
-    static let PageTextFont = FXFontStyles.Regular.caption1.scaledFont()
-    static let PageMarginScale = CGFloat(0.5)
-}
-
 class TabPrintPageRenderer: UIPrintPageRenderer {
-    fileprivate weak var tab: Tab?
-    let textAttributes = [NSAttributedString.Key.font: PrintedPageUX.PageTextFont]
+    private struct PrintedPageUX {
```

Comment:
sort of unrelated and totally non-blocking, but it seems standard to use just "UX" as the struct name